latex2lambda.py

```diff
@@ -0,0 +1,48 @@
+from latex2sympy2 import latex2sympy
+from sympy import lambdify
+
+
+class TexWithoutArgumentsError(Exception):
+    def __init__(self, msg: str):
+        super().__init__(msg)
+
+
+def latex2lambda(
+    tex: str,
+    variable_values={},
+    args=None,
+    modules=None,
+    printer=None,
+    use_imps=True,
+    dummify=False,
+):
+    if args is None:
+        try:
+            """
+            Trying to parse a function of type "f(...) = <...>" with the function itself on <...>
+            """
+            var_tex, tex = tex.replace(" ", "").split("=")
+            args = var_tex.split("(")[1].replace(")", "").split(",")
+        except Exception:
+            raise TexWithoutArgumentsError(
+                "No arguments provided. Either pass the arguments as list or as r'f(<args>) = <...>'."
+            )
```

Comment:
Consider this pseudo-code suggestion
```suggestion
        # Trying to parse a function of type "f(...) = <...>" with the function itself on <...>
        # 1. clean up white spaces
        tex = tex.replace(" ", "").replace("\t", "").replace("\n", "").replace("\r\n", "")
        # 2. get the two parts of the equation
        split_tex = tex.split("=")
        if len(split_tex) > 2:
            raise TexMalformattedError("expected function of type ... got  two equal signs '='")
        var_tex, tex = split_tex # could be done in a different way here, this doesn't feel too pythonic tbh
        if len(var_tex.split("(")) < 2:
            raise TexMalformattedError("expected function of type ... got  ...")
        # 3. pick up the args
        args = var_tex.split("(")[1].replace(")", "").split(",")
        # 4. double check some validity - and maybe throw a different exception so that the library user can better understand which part he failed.
        if len(args) < 1:
            raise TexNoFunctionArgs("expected function of type ... got  zero arguments")
        
```